__tests__/news.test.js

```diff
@@ -36,6 +36,7 @@ describe("/api/topics", () => {
         .then(({ body }) => {
           const topics = body;
```

Comment:
your data should be sent back as a key on body.  double check that you're running your tests before commiting.